### PR TITLE
Handle cases where target object is not yet created.

### PR DESCRIPTION
**Pull request by bhuvnesh2703**
This commit handles revert failure when the target cluster is not yet
created.

### CODE REVIEW

File: hub/revert.go [hunk 1]
```diff
@@ -165,7 +165,7 @@ func (s *Server) Revert(_ *idl.RevertRequest, stream idl.CliToHub_RevertServer)
 			return err
 		}
 
-		return ArchiveSegmentLogDirectories(s.agentConns, s.Config.Target.MasterHostname(), archiveDir)
+		return ArchiveSegmentLogDirectories(s.agentConns, s.Config.Source.MasterHostname(), archiveDir)
```
Comment (kalensk) [inline hub/revert.go:168]: Can you explain this change? Seems unrelated to this particular commit. Splitting out into a separate commit with explanation in message would be easier to follow.

Comment (bhuvnesh2703) [reply hub/revert.go:168]: i can add a comment if you want, but in this case as well target object is not yet created and instead of using Target Hostname we should rely on source hostname.

Comment (kalensk) [reply hub/revert.go:168]: I see. Thanks for the additional context. (I don't think a comment is needed. The BATs tests and other tests should catch this pretty quick if someone happens to change it back.)

**Review by kalensk — APPROVED** [review]
